mobile/ios/ArcadiaAssistant.xcodeproj/project.pbxproj

```diff
@@ -410,14 +410,14 @@
 		8070C9BF504549C88161DD60 /* RNVectorIcons.xcodeproj */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = "wrapper.pb-project"; name = RNVectorIcons.xcodeproj; path = "../node_modules/react-native-vector-icons/RNVectorIcons.xcodeproj"; sourceTree = "<group>"; };
 		832341B01AAA6A8300B99B32 /* RCTText.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = RCTText.xcodeproj; path = "../node_modules/react-native/Libraries/Text/RCTText.xcodeproj"; sourceTree = "<group>"; };
 		8C7B05C4BD90420B8B507F44 /* Entypo.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = Entypo.ttf; path = "../node_modules/react-native-vector-icons/Fonts/Entypo.ttf"; sourceTree = "<group>"; };
-		8F1FF9B020345FBE005D03F2 /* aa-iconfont.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "aa-iconfont.ttf"; sourceTree = "<group>"; };
+		8F467AB320385E91002D5924 /* aa-iconfont.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; name = "aa-iconfont.ttf"; path = "../../../assets/fonts/aa-iconfont.ttf"; sourceTree = "<group>"; };
 		97F364A02A7149BEB21F7974 /* SimpleLineIcons.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = SimpleLineIcons.ttf; path = "../node_modules/react-native-vector-icons/Fonts/SimpleLineIcons.ttf"; sourceTree = "<group>"; };
 		9D0E06A9EA5049CDB5E05544 /* Ionicons.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = Ionicons.ttf; path = "../node_modules/react-native-vector-icons/Fonts/Ionicons.ttf"; sourceTree = "<group>"; };
 		ADBDB91F1DFEBF0600ED6528 /* RCTBlob.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = RCTBlob.xcodeproj; path = "../node_modules/react-native/Libraries/Blob/RCTBlob.xcodeproj"; sourceTree = "<group>"; };
+		B20DFF6B416A4019B861B7CD /* aa-iconfont.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = "aa-iconfont.ttf"; path = "../assets/fonts/aa-iconfont.ttf"; sourceTree = "<group>"; };
```

Comment:
In that record i believe. but why is the top one? `8F467AB320385E91002D5924 /* aa-iconfont.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; name = "aa-iconfont.ttf"; path = "../../../assets/fonts/aa-iconfont.ttf"; sourceTree = "<group>"; };`